games/guess-it/server/routes/radix.js

```diff
@@ -0,0 +1,286 @@
+            // console.log("Response", {response});
+
+            io.to(component).emit("server", {message: "User joined the room..."});
+
+            const state = await check_state(component);
+            io.to(component).emit("got-state", state);
+        });
+
+        socket.on('make-guess', async (guess) => {
+            const user = USERS[socket.id];
+            const response = await make_guess(user.key, user.account, user.component, guess);
+            // console.log("Response", response, guess);
+            io.to(user.component).emit("server", {message: response});
+            
+            const state = await check_state(user.component);
+            io.to(user.component).emit("got-state", state);
+        });
+
+        socket.on('withdraw-funds', async () => {
+            const user = USERS[socket.id];
+            const response = await withdraw_funds(user.key, user.account, user.component);
+            // console.log("Response", response);
+            socket.emit("server", {message: response});
+            
+            const state = await check_state(user.component);
+            io.to(user.component).emit("got-state", state);
+        });
+        
+        socket.on('disconnect', () => {
+          console.log(`user '${socket.id} disconnected`);
+          delete USERS[socket.id];
+        });
+        
+        socket.emit("got-games", GAMES);
+    });
+
+    return function middleware(req, res, next) {
+        next();
+    }
+}
+
+    
+/* GET home page. */
+router.get('/', function(req, res, next) {
+    const games = Object.keys(GAMES).map(k => GAMES[k]);
+    console.log('games: ', {games});
+    res.render('radix', { title: 'Radix Guess-It Game', games });
+});
+
+router.post('/new-account', async function(req, res, next) {
+    const response = await new_account();
+    if (response.error){ return res.status(500).json(response) }
+    return res.json(response);
+});
+
+router.post('/login', async function(req, res, next) {
+    const { public, private } = req.body;
+    const response = await login_user(public, private);
+    if (response.error){ return res.status(500).json(response) }
+    return res.json(response.split("\n").slice(0,-1));
+});
+
+router.post('/create-game', async function(req, res, next) {
+    const { name, bet } = req.body;
+    const response = await create_game(name, bet);
+    if (response.error){ return res.status(500).json(response) }
+
+    const io = getIo(req);
+    io.emit("got-games", GAMES);
+
+    return res.json(response);
+});
+
+router.post('/join-game', async function(req, res, next) {
+    const { public, private, component, bet } = req.body;
+    const response = await join_game(public, private, component, bet);
+    if (response.error){ return res.status(500).json(response) }
+
+    return res.json(response);
+});
+
+router.post('/make-guess', async function(req, res, next) {
+    const { public, private, component, guess } = req.body;
```

Comment:
Likely the first 2 params could go since they are being stored on the server during io.connection phase